crates/transaction-pool/src/blobstore/disk.rs

```diff
@@ -52,22 +53,33 @@ impl BlobStore for DiskFileBlobStore {
 
     fn insert_all(&self, txs: Vec<(B256, BlobTransactionSidecar)>) -> Result<(), BlobStoreError> {
         if txs.is_empty() {
-            return Ok(())
+            return Ok(());
         }
         self.inner.insert_many(txs)
     }
 
     fn delete(&self, tx: B256) -> Result<(), BlobStoreError> {
-        self.inner.delete_one(tx)?;
+        let path = self.inner.blob_disk_file(tx);
+        self.files_to_delete.write().insert(path);
```

Comment:
let's only store hashes in the map and compute the paths when we need to delete them

Reply:
Not the smartest question, but by hashes do you mean using a standard hash function to hash the path name? Or do you mean just pushing it in the hashset?

Reply:
The transaction hash itself (`tx`) :)

Reply:
Gotcha, thanks! My bad! Just to be clear, we just have to clear out the files_to_delete instance, and not actually delete any files in that path. Did I get that right?

Reply:
Essentially the ask is: instead of `files_to_delete` being a `HashSet<PathBuf>`, it should be a `HashSet<B256>`. then, when we actually delete the files in `cleanup`, we resolve those hashes to a path using `self.inner.blob_disk_file(hash)`. Does that make sense? 😄

Reply:
Absolutely! I'm pretty new to Rust and this codebase, still get tripped up time to time! Thx.

Reply:
That makes sense! I'll need an hour to fix this, please ignore my latest push! Thx

Reply:
Ok, I made some changes. Let me know if I got this right! I also rebased this, and can squash commits later if needed when everything makes sense.